From 3ce287a083443ab0e32cda29a0aa3138d14d49f0 Mon Sep 17 00:00:00 2001
From: Marek Marczykowski <marmarek@invisiblethingslab.com>
Date: Wed, 17 Apr 2013 01:40:55 +0200
Subject: [PATCH 09/11] libxl: pass ipaddr to libxl toolstack
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
Organization: Invisible Things Lab
Cc: Marek Marczykowski-Górecki <marmarek@invisiblethingslab.com>

Do not silently ignore its value.

Signed-off-by: Marek Marczykowski-Górecki <marmarek@invisiblethingslab.com>
---
 src/libxl/libxl_conf.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/libxl/libxl_conf.c b/src/libxl/libxl_conf.c
index 71f5851..6ddcedf 100644
--- a/src/libxl/libxl_conf.c
+++ b/src/libxl/libxl_conf.c
@@ -1004,6 +1004,8 @@ libxlMakeNic(virDomainDefPtr def,
         case VIR_DOMAIN_NET_TYPE_ETHERNET:
             if (VIR_STRDUP(x_nic->script, l_nic->script) < 0)
                 return -1;
+            if (VIR_STRDUP(x_nic->ip, l_nic->data.bridge.ipaddr) < 0)
+                return -1;
             break;
         default:
             virReportError(VIR_ERR_INTERNAL_ERROR,
-- 
1.8.3.1

